### PR TITLE
Add ScrollChange binding

### PR DESCRIPTION
**Pull request by xfumihiro**

### CODE REVIEW

File: rxbinding/src/androidTest/java/com/jakewharton/rxbinding/view/RxViewTest.java [hunk 1]
```diff
@@ -260,6 +264,32 @@
     o.assertNoMoreEvents();
   }
 
+  @TargetApi(M)
+  @SdkSuppress(minSdkVersion = M)
+  @Test @UiThreadTest public void scrollChangeEvents() {
```
Comment (JakeWharton) [inline rxbinding/src/androidTest/java/com/jakewharton/rxbinding/view/RxViewTest.java]: This test fails locally when I run it against a 23 emulator. Do you see this?

```
com.jakewharton.rxbinding.view.RxViewTest > scrollChangeEvents[Nexus4_6.0(AVD) - 6.0] FAILED
	java.util.NoSuchElementException: No event found while waiting for OnNext
	at com.jakewharton.rxbinding.RecordingObserver.takeEvent(RecordingObserver.java:41)
```

Comment (xfumihiro) [inline rxbinding/src/androidTest/java/com/jakewharton/rxbinding/view/RxViewTest.java]: Yeah, sorry about that. I haven't run it on local M-emulator yet. Just start downloading the emulator image on a hacked wifi, might take a while...

Comment (JakeWharton) [inline rxbinding/src/androidTest/java/com/jakewharton/rxbinding/view/RxViewTest.java]: I fixed it. Merging manually. Probably was that the view starts at 0,0 and ignores scrolls to the same position as the current one.

**Comment by JakeWharton** [comment]
Merged manually with test fix.

**Comment by JakeWharton** [comment]
b541ee839718c9a385f2b47f5cd673c37fcfe4e7

**Comment by xfumihiro** [comment]
Thanks!

**Pull request closed without merging**